src/Tools/ExternalAccess/Razor/IRazorSpanMappingService.cs

```diff
@@ -13,5 +13,6 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
     internal interface IRazorSpanMappingService
     {
         Task<ImmutableArray<RazorMappedSpanResult>> MapSpansAsync(Document document, IEnumerable<TextSpan> spans, CancellationToken cancellationToken);
+        Task<ImmutableArray<RazorMappedEditoResult>> MapTextChangesAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken);
```

Comment:
Should we create a new interface to include this new method (`IRazorSpanAndEditMappingService`?) to avoid needing a dual insertion?

Reply:
Yea I'm definitely not opposed to that